apps/testing-javascript/src/path-to-purchase/pricing.tsx

```diff
@@ -437,7 +466,7 @@ export const PriceDisplay = ({status, formattedPrice}: PriceDisplayProps) => {
     (formattedPrice?.unitPrice || 0) * (formattedPrice?.quantity || 0)
 
   const percentOff = appliedMerchantCoupon
-    ? Math.floor(appliedMerchantCoupon.percentageDiscount * 100)
+    ? Math.floor(+appliedMerchantCoupon.percentageDiscount * 100)
```

Comment:
The `+` coerces `Prisma.Decimal` to a `number` so that regular arithmetic like `*` works.